tests/unit/PhpAssumptions/CliTest.php

```diff
@@ -28,4 +28,18 @@ public function itShouldShowHelpWithNoArgs()
 
         $this->assertTrue(strpos($contents, 'Usage: phpa') !== false);
     }
+
+    /**
+     * @test
+     */
+    public function itShouldTraverseADirectoryIfTargetIsADirectory()
+    {
+        ob_start();
+        $this->cli->handle([dirname(__FILE__) . '/../../fixtures']);
```

Comment:
We should probably introduce a `FIXTURES_DIR` constant in `tests/bootstrap.php` to stay away from duplication :)